test/fg/flagsmith_realtime_test.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flagsmith/flagsmith.dart';
+import 'package:flutter_client_sse/flutter_client_sse.dart';
+import 'package:http_mock_adapter/http_mock_adapter.dart';
+import 'package:test/test.dart';
+
+import '../shared.dart';
+
+const userIdentifier = 'test__user';
+
+void main() {
+  group('[Realtime updates]', () {
+    late FlagsmithClient fs;
+    late DioAdapter _adapter;
+    setUp(() async {
```

Comment:
Can we put a bit of whitespace before each of the sections & tests to make it a bit more readable